scripts/nixos-common-styles-dev-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if (( $# != 1 )); then
+	echo "Usage: $0 <path to local checkout>"
+	exit 1
+fi
+
+local_checkout="$1"; shift
+
+echo " :: Setting up development environment to use a local NixOS common styles checkout."
+
+if [ -e site-styles/common-styles ]; then
+	echo -n "Removing possibly stale symlink: "
+	rm -vf site-styles/common-styles
+fi
+
+echo -n "Linking with local checkout: "
+ln -vs "$(readlink -f "$local_checkout/src")" site-styles/common-styles
```

Comment:
I wonder if we could do this just in the `serve` command. And instead of symlinking we would have a variable in `Makefile`. Then on a reload we would run `make COMMON_STYLES_DIR=...`.